h/static/scripts/annotator/selections.js

```diff
@@ -1,13 +1,28 @@
 'use strict';
 
+var Observable = require('zen-observable');
```

Comment:
Requiring both `Observable` and `observable` in the same file is pretty ugly. It looks like `../util/observable` already exposes the constructor so you could just use `observable.Observable.of` if necessary...?